### PR TITLE
[AUT-3670] fix: return hb special symbols around placeholder for proper saving

### PR DESCRIPTION
**Pull request by KirylHatalski**
Ticket: [AUT-3670](https://oat-sa.atlassian.net/browse/AUT-3670)

## What's Changed

- Fixed the saving of item interaction when it have curly braces around placeholder

## Demo
image

## Env
http://test-kiril.playground.kitchen.it.taocloud.org:40351/tao/Main/login
admin/admin

reguire merge of https://github.com/oat-sa/tao-item-runner-qti-fe/pull/404 first

[AUT-3670]: https://oat-sa.atlassian.net/browse/AUT-3670?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: views/js/qtiCreator/widgets/item/Widget.js [hunk 1]
```diff
@@ -135,7 +135,9 @@ define([
                 reject(err);
             }
 
-            const xml = xmlNsHandler.restoreNs(xmlRenderer.render(this.element), this.element.getNamespaces(), true);
+            const xml = xmlNsHandler
+                .restoreNs(xmlRenderer.render(this.element), this.element.getNamespaces(), true)
+                .replace('&lcub;', '{');
```
Comment (jsconan) [inline views/js/qtiCreator/widgets/item/Widget.js:140]: **question:** Is this change actually needed? The transformed braces would be part of the user's defined content, and it will display properly in the end. Unless this is an issue with the XML parser and the QTI validator?

**Review by jsconan — COMMENTED** [review]
I'm not sure about the need of replacing back entities to the original charcode

**Review by jsconan — APPROVED** [review]
**Review Checklist**
- [ ] New code is covered by tests (if applicable)
- [x] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [X] New code respects code style rules
- [x] New code respects best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Pull request's target is not `master`
- [x] Commits are following [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] Commits messages are meaningful
- [x] Commits are atomic

**Review by viktar-dzmitryieu-tao — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [X] New code is respecting code style rules
- [X] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [X] Acceptance criteria are respected
- [X] Pull request title and description are meaningful
- [X] Pull request's target is not `master`

**Review by Karol-Stelmaczonek — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
